assets/component-image-with-text.css

```diff
@@ -327,3 +327,17 @@
     margin-left: 0;
   }
 }
+
```

Comment:
> looks like Medium height doesn’t work, as I obtain a super short height.

There's a note about this in the PR description. Medium won't work until https://github.com/Shopify/dawn/pull/2148 merges

> Can we align content on the grid? If that complicated no need to do it.

If we want to introduce the additional magic for it, then technically yes. I just wasn't sure if we were aligned to do that or not. I can add it, but we should probably align as a group on that.

@YoannJailin

Reply:
@YoannJailin padding magic has been added. Intended logic is as follows

Removes..
- left padding on image-second rows
- right padding on image-first rows
- both left/right padding on mobile

If..
- both color scheme settings match, and..
- content container has no visible border, and..
- content container has no visible shadow

Reply:
Yeah basically. This is what I'm seeing with the padding removed but text centered. It just looks a little odd to me
<img src="https://screenshot.click/20-13-ylj3b-a1sro.png" width="250"/>

Reply:
@kmeleta I agree! Let's remove the magic on desktop when content is centered then :)